src/models/UserMapper.php

```diff
@@ -38,6 +38,7 @@ public function getVerboseFields()
             "username"         => "username",
             "full_name"        => "full_name",
             "twitter_username" => "twitter_username",
+            "trusted"          => "trusted"
```

Comment:
I'm just thinking of handling being admin and being trusted the same way. 

Currently the "trusted" state can be seen by *any* user GETing the users information. But the information of being an admin is not available.

On the other hand, exposing that information only when the user itself is retrieving it is something completely different…